### PR TITLE
feat: Add FlexibleImpactNumberCards

### PR DESCRIPTION
**Pull request by aprigge**
Connected to [APPS-1918](https://jira.library.ucla.edu/browse/APPS-1918)

https://deploy-preview-197--ucla-library-storybook.netlify.app/?path=/story/flexible-impact-number-cards--default

### CODE REVIEW

**Review by dianneweinthal — CHANGES_REQUESTED** [review]
edits in comments - let me know if you have any questions!

File: src/lib-components/ImpactNumberCard.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        color: var(--color-primary-blue-05);
+        text-align: center;
+    }
+
+    .text {
+        padding-top: 16px;
+        @include step-0;
+        color: var(--color-primary-blue-05);
+    }
+}
```
[the remark this answers is not in the document]
Comment (dianneweinthal) [reply src/lib-components/ImpactNumberCard.vue:82]: my bad, we can remove this!

**Review by pghorpade — APPROVED** [review]
👍

**Review by dianneweinthal — DISMISSED** [review]
👍

**Review by dianneweinthal — APPROVED** [review]
👍

**Pull request merged**